### ISSUE TITLE
I am not familiar with Ray

### ISSUE THREAD

**Issue by guox18** [volcengine/verl#345]
i wonder how can i set CUDA_VISIBLE_DEVICES properly? (if i set CUDA_VISIBLE_DEVICES  in my environment and run the training script, there is just no use)

**Comment by PeterSH6** [comment]
Hi @guox18 , just wondering why you need to set CUDA_VISIBLE_DEVICES when running the scripts?
And which script are you running with?

**Comment by kevin85421** [comment]
@PeterSH6, would you mind adding a "ray" label to this issue? I am triaging Ray-related issues in veRL. Thanks!

**Comment by PeterSH6** [comment]
@kevin85421 Done

**Comment by kevin85421** [comment]
Hi @guox18, I don't know how does veRL launch a Ray cluster, but there are two methods:

* Method 1: `os.environ["CUDA_VISIBLE_DEVICES"] = "1,2,3"`
  ```python
  import os
  import ray
  import torch
  
  from ray.util.placement_group import placement_group
  from ray.util.scheduling_strategies import PlacementGroupSchedulingStrategy
  
  os.environ["CUDA_VISIBLE_DEVICES"] = "1,2,3"
  ray.init()
  
  pg = placement_group([{"GPU": 1}, {"GPU": 1}, {"GPU": 1}])
  ray.get(pg.ready(), timeout=10)
  
  @ray.remote(num_gpus=1, num_cpus=0)
  def f():
      assert torch.cuda.device_count() == 1
      return os.environ["CUDA_VISIBLE_DEVICES"]
  
  
  # Create an actor to a placement group.
  tasks = [
      f.options(
          scheduling_strategy=PlacementGroupSchedulingStrategy(
              placement_group=pg,
          )
      ).remote()
      for _ in range(3)
  ]
  
  print(ray.get(tasks))
  # [Example output]:
  # 2025-03-03 02:09:59,645 INFO worker.py:1832 -- Started a local Ray instance. View the dashboard at 127.0.0.1:8265
  # ['2', '3', '1'] --> GPU 0 is not used.
  ```

* Method 2:
  * `CUDA_VISIBLE_DEVICES=1,2,3 ray start --head --num-gpus=3`: Launch a Ray node with 3 GPUs (GPU1, GPU2, GPU3)
  * `python3 test.py`: Submit Ray tasks to the existing Ray cluster.
    ```python
    import os
    import ray
    import torch
    
    from ray.util.placement_group import placement_group
    from ray.util.scheduling_strategies import PlacementGroupSchedulingStrategy
    
    ray.init()
    
    pg = placement_group([{"GPU": 1}, {"GPU": 1}, {"GPU": 1}])
    ray.get(pg.ready(), timeout=10)
    
    @ray.remote(num_gpus=1, num_cpus=0)
    def f():
        assert torch.cuda.device_count() == 1
        return os.environ["CUDA_VISIBLE_DEVICES"]
    
    
    # Create an actor to a placement group.
    tasks = [
        f.options(
            scheduling_strategy=PlacementGroupSchedulingStrategy(
                placement_group=pg,
            )
        ).remote()
        for _ in range(3)
    ]
    
    print(ray.get(tasks))

    # [Example output]
    # 2025-03-03 02:13:26,938 INFO worker.py:1654 -- Connecting to existing Ray cluster at address: 172.31.9.244:6379...
    # 2025-03-03 02:13:26,947 INFO worker.py:1832 -- Connected to Ray cluster. View the dashboard at 127.0.0.1:8265
    # ['3', '1', '2']
    ```
